client/network/src/service.rs

```diff
@@ -916,6 +949,17 @@ impl Metrics {
 			peerset_num_requested: register(Gauge::new(
 				"sub_libp2p_peerset_num_requested", "Number of nodes that the peerset manager wants us to be connected to",
 			)?, registry)?,
+			pending_connections: register(Gauge::new(
+				"sub_libp2p_pending_connections",
+				"Number of connections in the process of being established",
+			)?, registry)?,
+			pending_connections_errors_total: register(CounterVec::new(
+				Opts::new(
+					"sub_libp2p_pending_connections_errors_total",
+					"Total number of failures to connection to a node"
```

Comment:
```suggestion
					"Total number of node connection failures"
```

Just a suggestion.

Reply:
Or just "Total number of pending connection errors."